schedule/btrfs_ipmi.yaml

```diff
@@ -0,0 +1,26 @@
+name:           btrfs@64bit-ipmi
+description:    >
+      testik
```

Comment:
testik ?

Reply:
ofc, it was a test :D

Reply:
And it was a small version of the original one. It makes a perfect sense to call it "testik"